src/video/video_source_local.rs

```diff
@@ -221,6 +242,7 @@ impl VideoSource for VideoSourceLocal {
                     }
                 }
             }
+
```

Comment:
Avoid adding style changes on the same commit as function changes. No need to change on this one, but be aware of future commits. `git add -p` or `git checkout -p` is your friend

Reply:
Sure, I can remove it, I'll be doing a fix commit anyway to change the technique to sort->dedup.